README.md

```diff
@@ -982,7 +982,7 @@
     // declaration to the top of the scope,
     // which means our example could be rewritten as:
     function example() {
-      let declaredButNotAssigned;
+      var declaredButNotAssigned;
```

Comment:
Correct me if I'm wrong. Based on the comment on line 981 -  `The interpreter is hoisting the variable declaration to the top of the scope, which means our example could be rewritten as`. It seems that the code from the example above would be rewritten in the perspective of interpreter, which shows the `var` will be hoisted to the top of the block.

I think that `let` isn't hoisted is showing on the line 992.